src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,39 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.academy");
 
+    public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        // Create
+        Book book2 = new Book();
+        book2.setTitle("breakfast for champions");
+        book2.setPrice(BigDecimal.valueOf(28));
+        System.out.println(bookDao.create(book2));
+        // Read
+        System.out.println(bookDao.findAll());
+        long id = 1L;
+        Optional<Book> bookOptional = bookDao.findById(id);
```

Comment:
Id must be generated by the database. Do not hardcode it in the code
```suggestion
        Optional<Book> bookOptional = bookDao.findById(book2.getId());
```